python/lsst/pipe/tasks/gen2measureExtendedPsf.py

```
@@ -0,0 +1,187 @@
+# This file is part of pipe_tasks.
```

I'm confused about the addition of this file in the context of this ticket?

---

Oh yeah, that's just me failing hard at git. Removed it. Sorry!